src/commands/server/restore.ts

```diff
@@ -85,10 +115,208 @@ export default class Restore extends Command {
   private getRestoreTasks(flags: any): Listr.ListrTask[] {
     return [
       {
-        title: 'Scheduling restore...',
+        title: 'Detecting existing operator version...',
+        enabled: () => flags.version || flags.rollback || flags['backup-cr'],
```

Comment:
I think we can always detect operator version.

Reply:
We don't have to. If there is no request to redeploy operator with a different version - then it doesn't make sense for us now.

Reply:
If it doesn't break anything then remove.

Reply:
ok, but we'll do work which is not needed in some cases. The condition regulates when we need this info and when we don't